src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -35,6 +35,19 @@
 
   <!-- For CPS/VS support. Importing in .props allows any subsequent targets to redefine this if needed -->
   <Target Name="CompileDesignTime" />
+    
+    <Target Name="SFCustomizePacking" AfterTargets="CoreBuild">
```

Comment:
this will add the single project dacpac or all dacpacs in the build folder to be added to the nuget package automatically.